modules/@apostrophecms/piece-type/index.js

```diff
@@ -1060,6 +1060,94 @@ module.exports = {
+            throw new Error('Localized option not set to true, so the module cannot be localized.');
+          }
+
+          console.log('Adding drafts and locales to documents');
+
+          const locales = Object.keys(self.apos.i18n.locales);
+          const lastPublishedAt = new Date();
+          const inserts = [];
+          const deletes = [];
+
+          await self.apos.migration.eachDoc({ type: self.name }, async doc => {
+            if (doc.aposDocId && !doc._id.endsWith('published') && !doc._id.endsWith('draft')) {
+              deletes.push(doc._id);
+
+              for (const locale of locales) {
+                const newDraft = {
+                  ...doc,
+                  aposLocale: `${locale}:draft`,
+                  aposMode: 'draft',
+                  aposDocId: doc._id,
+                  _id: `${doc.aposDocId}:${locale}:draft`
+                };
+                const newPublished = {
+                  ...doc,
+                  aposLocale: `${locale}:published`,
+                  aposMode: 'published',
+                  aposDocId: doc._id,
+                  _id: `${doc.aposDocId}:${locale}:published`,
+                  lastPublishedAt
+                };
+                inserts.push(newDraft);
+                inserts.push(newPublished);
+              }
+            }
+          });
+
+          await self.apos.doc.db.deleteMany({ _id: { $in: deletes }});
+          await self.apos.doc.db.insertMany(inserts);
+          await self.apos.attachment.recomputeAllDocReferences();
+
+          console.log(`Done localizing module ${self.name}`);
+        }
+      },
+
+      unlocalize: {
+        usage: 'Remove duplicate documents when a module has not "localized" and "autopublish" anymore.' +
+        '\nOptions are:' +
+        '\n- locale: if not set, it is the project\'s default locale' +
+        '\n- mode: by default, published' +
+        '\nExample: node app [moduleName]:unlocalize --mode=published --locale=en',
+        async task(argv) {
+          if (self.options.localized) {
+            throw new Error('Localized option not set to false, so the module cannot be unlocalized.');
+          }
+
+          const locale = argv.locale || self.apos.i18n.defaultLocale;
+          const mode = argv.mode || 'published';
+          const inserts = [];
+          const deletes = [];
+
+          console.log(`Removing duplicated documents and updating ${mode} ones`);
+
+          await self.apos.migration.eachDoc({ type: self.name }, async doc => {
+            deletes.push(doc._id);
+            if (doc.aposDocId && doc.aposLocale === `${locale}:${mode}` && doc.aposMode === mode) {
+              const newDoc = {
+                ...doc,
+                aposLocale: undefined,
+                aposMode: undefined,
+                _id: doc.aposDocId
+              };
+              inserts.push(newDoc);
+            }
+          });
+
+          await self.apos.doc.db.deleteMany({ _id: { $in: deletes }});
```

Comment:
For this one, it's best not to do insertMany/deleteMany, for the reason I mentioned above.